desc/compute/_geometry.py

```diff
@@ -232,23 +212,48 @@ def _A_of_z_FourierRZToroidalSurface(params, transforms, profiles, data, **kwarg
     label="A",
     units="m^{2}",
     units_long="square meters",
-    description="Average cross-sectional area",
+    description="Average enclosed cross-sectional area, scaled by max(ρ)⁻²",
```

Comment:
This is a simple toroidal average in the sense of $A_0 = \int_{\zeta} A(\zeta) d\zeta / (2 \pi)$ matching the convention that `R0` is the average in the sense  $R_0 = \int_{\zeta} R(\rho=0) d\zeta / (2 \pi)$. This is different than the average value over the device. That would be $\langle R \rangle =  \int_{\zeta} R \Vert e_{\zeta}\Vert(\rho=0) d \zeta / \int_{\zeta}  \Vert e_{\zeta}\Vert(\rho=0) d \zeta$.